applications/ULFapplication/custom_processes/calculate_curvature.h

```diff
@@ -267,7 +272,8 @@ namespace Kratos
 		im->FastGetSolutionStepValue(MEAN_CURVATURE_2D) = curv_tp;			
 	      }
 	      
-	      if(im->FastGetSolutionStepValue(IS_STRUCTURE) != 0.0 && im->FastGetSolutionStepValue(TRIPLE_POINT)*1.0E8 == 0.0)
+ 	      if(im->FastGetSolutionStepValue(IS_STRUCTURE) != 0.0 && im->FastGetSolutionStepValue(TRIPLE_POINT)*1.0E8 == 0.0)
```

Comment:
I recommend to use flags, is faster and you don't need an extra variable

Reply:
Greetings dear @loumalouomega and very good day to you,

I would like to get your kind recommendation regarding using/defining flags locally,

As far as I am concerned, in our first pull request, when we were adding this file, https://github.com/KratosMultiphysics/Kratos/pull/1410, a similar flag issue was raised, and at that time it was not convenient to define our new flags in the core and I received the following feedback accordingly:

"If you don't think they are convenient for you, then just keep the current implementation. I also agree that adding a new local flag is not a good solution in this case"

so my concern here please, in the case now the flag variables can be defined locally, would you kindly please recommend me to follow a specific example/standard where these flags were defined and used locally.

thanks and best regards,

Reply:
The list of flags you can use is here:

https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/kratos_flags.h

If you need a new one you need to propose in an independent PR. The flags are limited to 64 and we can not add them happily. The flags you can use, according to you code could be:

FLUID
BOUNDARY
INTERFACE
FREE_SURFACE

Reply:
The only one you need to be careful is with ACTIVE. The nodes and conditions set as non ACTIVE will not be computed in the core builder and solvers